hasura/metadata/databases/default/tables/public_users.yaml

```diff
@@ -148,13 +148,25 @@ select_permissions:
           - deleted_at:
               _is_null: true
           - circle:
-              _and:
-                - deleted_at:
-                    _is_null: true
-                - users:
-                    profile:
-                      id:
-                        _eq: X-Hasura-User-Id
+              deleted_at:
+                _is_null: true
+          - _or:
+              - circle:
+                  users:
+                    _and:
+                      - deleted_at:
```

Comment:
a bit pedantic,but I've been putting the `deleted_at` clause as second after the profile check consistently for easier scanning and readability